### PR TITLE
i18n(fr): create error code references part 2

### PR DESCRIPTION
**Pull request by cravend**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Translated:
  - markdown errors

To do:
- Astro errors

Done:
- misc. errors: #3386

Note: this PR doesn’t depend on part 1 in any way, I’m just splitting it for ease of review

### CODE REVIEW

**Review by thomasbnt — APPROVED** [review]
LGTM! 🌱

**Review by dreyfus92 — APPROVED** [review]
LGTM! ✅

File: src/content/docs/fr/reference/errors/invalid-frontmatter-injection-error.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+> **InvalidFrontmatterInjectionError** : Un plugin remark ou rehype a tenté d'injecter du frontmatter invalide. Assurez-vous que "astro.frontmatter" est configuré comme un objet JSON valide qui n'est pas `null` ou `undefined`. (E06003)
+
+## Qu'est-ce qui a mal tourné ?
+
+Un plugin remark ou rehype a tenté d'injecter du frontmatter invalide. Cela se produit lorsque "astro.frontmatter" est `null`, `undefined` ou un objet JSON invalide.
+
+**Voir aussi :**
+
+-  [Modifier le frontmatter par un programme](/fr/guides/markdown-content#modifier-le-frontmatter-par-un-programme)
```
Comment (Yan-Thomas) [inline src/content/docs/fr/reference/errors/invalid-frontmatter-injection-error.mdx:15]:
```suggestion
- [Modifier le frontmatter par un programme](/fr/guides/markdown-content/#modifier-le-frontmatter-par-un-programme)
```

File: src/content/docs/fr/reference/errors/config-legacy-key.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+> **ConfigLegacyKey** : Configuration héritée détectée : `LEGACY_CONFIG_KEY`. (E07002)
+
+## Qu'est-ce qui a mal tourné ?
+
+Astro a détecté une option héritée de configuration dans votre fichier de configuration.
+
+**Voir aussi :**
+
+-  [Référence de configuration](/fr/reference/configuration-reference#options-h%C3%A9rit%C3%A9es)
```
Comment (Yan-Thomas) [inline src/content/docs/fr/reference/errors/config-legacy-key.mdx:15]:
```suggestion
- [Référence de configuration](/fr/reference/configuration-reference/#options-héritées)
```

**Review by Yan-Thomas — APPROVED** [review]
Thanks @cravend (and everyone for the review), great work! Fixed a few links in this PR, hope you don't mind. Welcome to Team Docs 🥳!

**Pull request merged**